content/en/docs/concepts/architecture/master-node-communication.md

```diff
@@ -104,6 +104,17 @@ all traffic destined for a kubelet, node, pod, or service through the tunnel.
 This tunnel ensures that the traffic is not exposed outside of the network in
 which the nodes are running.
 
-SSH tunnels are currently deprecated so you shouldn't opt to use them unless you know what you are doing. A replacement for this communication channel is being designed.
+SSH tunnels are currently deprecated so you shouldn't opt to use them unless you know what you are doing. APIServer network proxy is a replacement for this communication channel.
+
+### APIServer network proxy (alpha)
+
+As a replacement to the SSH tunnels, Kubernetes supports APIServer network proxy
+to protect the Master -> Cluster communication paths. The feature is still
```

Comment:
```suggestion
to protect the control plane → cluster communication. The feature is still
```
If you use a feature state shortcode, you don't need to repeat that detail in the text.

Reply:
What do you mean by "feature state shortcode"?

Reply:
Ah, sorry. The heading `### APIServer network proxy (alpha)` repeats the state from the [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state).

There's another problem: headings generate anchor URLs, and we plan to graduate this feature to GA in the future. So we should avoid making a URL that has “alpha” in it.